sound/soc/sof/topology.c

```diff
@@ -1860,6 +1874,7 @@ static int sof_route_load(struct snd_soc_component *scomp, int index,
 static int sof_route_unload(struct snd_soc_component *scomp,
 			    struct snd_soc_dobj *dobj)
 {
+	/* TODO: unload routes when topology is changed */
```

Comment:
is this something we are planning? In what cases would we have a dynamic topology change?

Reply:
yes, low memory and software development use cases. A lot of code is already in place for both FW and drivers, needs someone to complete + test